stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -82,6 +82,43 @@ __swift_uint32_t _swift_stdlib_cxx11_mt19937(void);
 SWIFT_RUNTIME_STDLIB_INTERFACE
 __swift_uint32_t
 _swift_stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound);
+  
+// Math library functions
+SWIFT_RUNTIME_STDLIB_INTERFACE float _swift_remainderf(float, float);
```

Comment:
Could you use the `_swift_stdlib_*` namespace for stdlib shims?  `_swift_*` is the general "Swift runtime" namespace.